mapie/tests/test_calibration.py

```diff
@@ -106,6 +106,18 @@
     X_, y_, test_size=0.33, random_state=random_state
 )
 
+gb = GradientBoostingClassifier(
+            random_state=random_state
+            )
```

Comment:
```suggestion
gb = GradientBoostingClassifier(random_state=random_state)
```

Reply:
I also suggest moving it directly into the test (before instantiating the `mapie` estimator) to better understand where `gb` is being used.